LFS171x/docs/introduction-to-hyperledger.md

```diff
@@ -1 +1,386 @@
+
+The enterprise continues to be at the heart of this course.
+
+## Why Businesses Choose to Use Hyperledger? (Brian Behlendorf)
+[![Why Businesses Choose to Use Hyperledger? (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/KX54mejdaOo)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+# Hyperledger Frameworks
+
+## Incubated Hyperledger Projects (Brian Behlendorf)
+In the following video, Brian Behlendorf covers the projects that are part of Hyperledger (as of October 2017). These include the Hyperledger frameworks, which are detailed in this section, as well as the Hyperledger modules, which are covered in the next section.
+
+[![Incubated Hyperledger Projects (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/eeZzxcdYNh8)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+## Components of Hyperledger Frameworks
+Hyperledger business blockchain frameworks are used to build enterprise blockchains for a consortium of organizations. They are different than public ledgers like the Bitcoin blockchain and Ethereum. The Hyperledger frameworks include:
+
+* An append-only distributed **ledger**
+* A **consensus algorithm** for agreeing to changes in the ledger
+* **Privacy** of transactions through permissioned access
+* **Smart contracts** to process transaction requests.
+
+![Components of Blockchain for Business](../images/introduction-to-hyperledger/Components_of_blockchain.jpg)
+
+Now, let's explore the five Hyperledger frameworks (as of October 2017)!
+
+## Hyperledger Iroha v0.95
+[Hyperledger Iroha](https://hyperledger.org/projects/iroha) is a blockchain framework contributed by Soramitsu, Hitachi, NTT Data, and Colu. Hyperledger Iroha is designed to be simple and easy to incorporate into infrastructure projects requiring distributed ledger technology. Hyperledger Iroha emphasizes mobile application development with client libraries for Android and iOS, making it distinct from other Hyperledger frameworks. Inspired by Hyperledger Fabric, Hyperledger Iroha seeks to complement Hyperledger Fabric and Hyperledger Sawtooth, while providing a development environment for C++ developers to contribute to Hyperledger.
+
+In conclusion, Hyperledger Iroha features a simple construction, modern, domain-driven C++ design, along with the consensus algorithm [YAC](https://www.overleaf.com/read/wzhwjzbjvrzn#/40115559/).
+
+## Introduction to Hyperledger Sawtooth (Courtesy of Sawtooth)
+[![Introduction to Hyperledger Sawtooth (Courtesy of Sawtooth)](../images/video-image.png)](https://youtu.be/Buw3g8oNG74)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
```

Comment:
Remove - the course is actually licensed under https://creativecommons.org/licenses/by/4.0/. Unfortunately, EdX doesn't let you specify which Creative Common license for videos. Therefore, we will let the overall license cover these.